user-guide/live-traffic/web-sessions/web-sessions-list.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Web Sessions List
+slug: web-sessions-list
+tags: web sessions, capturing traffic, live traffic
+publish: true
+position: 30
+---
+
+# Web Sessions List
```

Comment:
The "Save", "Share" and "Remove All" sections seem to be identical to the ones in Web Sessions Toolbar (a bit more extensively explained). Consider keeping those only in one place (if needed you can link between the two articles). Also, consider having only one article on the Live Traffic if the info overlaps so much.
- You have already added a link between the articles, but more of the info still repeats and the only difference is "the button" vs "the menu".

Reply:
We will sync on that at a later point - keeping both articles for now as the main idea was to cover the whole UI